test/throughput/test_collector_profile.py

```diff
@@ -0,0 +1,129 @@
+import gym
+import numpy as np
+import pytest
+from gym.spaces.discrete import Discrete
+from gym.utils import seeding
+
+from tianshou.data import Batch, Collector, ReplayBuffer
+from tianshou.env import VectorEnv
```

Comment:
Also add subprocVenv to test. I think it should be in another python test “test_env.py”